tachyon/zk/plonk/expressions/evaluator/simple_selector_extractor_unittest.cc

```diff
@@ -16,16 +16,17 @@ TEST_F(SimpleSelectorExtractorTest, Constant) {
   GF7 value = GF7::Random();
   std::unique_ptr<Expression<GF7>> expr =
       ExpressionFactory<GF7>::Constant(value);
-  EXPECT_FALSE(ExtractSimpleSelector(expr.get()).has_value());
+  EXPECT_FALSE(ExtractSimpleSelector<GF7>(expr.get()).has_value());
```

Comment:
Please remove `<GF7`>, since it can be inferenced.

```suggestion
  EXPECT_FALSE(ExtractSimpleSelector(expr.get()).has_value());
```